modules/system/system.block.ini

```diff
@@ -76,12 +76,12 @@ scale: 0.001494582
 type: read
 description: PLL locked for SMA external clock
 
-[EXT_CLOCK]
+[CLOCK_SOURCE]
 type: param enum
 description: External sma and event receiver clock enables
 0: int clock
 1: sma clock
-2: event receiver
+2: mgt recovered clock
```

Comment:
Change from mgt recovered clock to sfp1 recovered clock

Reply:
Done!